CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- added: Support the JavaScript `Set` and `Map` types in serialized data.
```

Comment:
While we won't do this in Edge, since this is a general purpose lib, it's worth noting to users that Set and Map in yaob don't properly support non-primitive keys such as arrays and objects. Since such keys work by reference, they will never match since it's created by yaob, never by the client or server.

Reply:
If you do put objects in a `Set` or `Map`, the yaob bridging will "work" in the sense that the data appears correctly.

We already accept that `===` doesn't work quite right for remote objects, but this isn't a new problem for sets or maps. It's true that these structures rely on `===` if you try to mutate them, but mutating remote objects has always been a bad idea, since the bridge will just stomp them on the next update.

So I don't think there are actually any new caveats here. The core (or any other user) can create sets and maps with objects as keys, send them over, and not worry. On the core side, mutation works correctly, and on the remote side, mutation is just as broken as it always has been.